blaze-client/src/test/scala/org/http4s/client/blaze/BlazePooledHttp1ClientSpec.scala

```diff
@@ -1,7 +1,31 @@
-package org.http4s.client.blaze
+package org.http4s
+package client
+package blaze
 
-import org.http4s.client.ClientRouteTestBattery
+import scalaz.concurrent.Task
 
+class BlazePooledHttp1ClientSpec
+  extends { val client = PooledHttp1Client() }
+    with ClientRouteTestBattery("Blaze PooledHttp1Client", client) {
 
-class BlazePooledHttp1ClientSpec extends ClientRouteTestBattery("Blaze PooledHttp1Client", PooledHttp1Client())
+  val path = "/simple"
```

Comment:
The only weird part of that is that we are testing our test suite rather than testing the software then.

So comment for us knowing whats going on or some more standardized client testing would be a nice addition but I feel this is definitely a good start that can be added to.